rust/src/bin/test-mobile.rs

```diff
@@ -0,0 +1,72 @@
+extern crate tracing;
```

Comment:
I think we should start putting those in a `rust/examples` directory. The `src/bin` directory should contain code we don't intend to ship.